tools/project_manager/diff.php

```diff
@@ -169,6 +186,9 @@ function do_navigation($projectid, $image, $L_round_num, $R_round_num, $L_user_c
     $navigation_text .= "\n<input type='hidden' name='L_round_num' value='$L_round_num'>";
     $navigation_text .= "\n<input type='hidden' name='R_round_num' value='$R_round_num'>";
     $navigation_text .= "\n<input type='hidden' name='format' value='$format'>";
+    if ($bb_diffs) {
```

Comment:
I may be wrong, but does that mean the URLs linked to in the navigation buttons will then always have the `bb_diffs` parameter with its value either on or off? A reason for not doing that is that general visibility of this feature is restricted to mentor-types. I think the way I currently have it means a non-mentor won't see that parameter in the URL when they use a button like `Proofreader Next`. Sorry if I've misunderstood what you are suggesting.

Reply:
I wouldn't worry about it being present for non-mentors since it won't have any impact and it will simplify this form code some. Just my opinion, may want to see what others say.

Reply:
I thought about this too, but I think Nigel's approach of not having the param for users who can't see it is the better approach IMHO. Because one of our eagle-eyed users is going to see this param and then try changing it and ask about it, and while it doesn't reveal anything they *can't* see, it's just more hassle than hiding it to begin with.